packages/components/src/SectionMessage/SectionMessage.tsx

```diff
@@ -74,6 +75,14 @@ export interface SectionMessageProps {
    * Adds a close button, makes the section message dismissable.
    */
   closeButton?: boolean;
+  /**
+   * Handler that is called when you need to control the dismissable message to close.
+   */
+  onClose?: () => void;
```

Comment:
should we do a `onCloseChange` here so you can have both (open/close) like in the dialog?

Reply:
you mean rename it?

Reply:
open would not make sense to me, how can you open a section message?